README.md

```diff
@@ -158,7 +158,10 @@ If you need some inspiration, [start with this](https://www.youtube.com/watch?v=
 
 ### Python
 
+- [What Does It Take To Be An Expert At Python?](https://www.youtube.com/watch?v=7lmCu8wz8ro&t=1927s) - by _James Powell_
 - [10 Tips for Pythonic Code](https://www.youtube.com/watch?v=_O23jIXsshs) - by _Michael Kennedy_
+- [7 Simple Tricks to Write Better Python Code](https://www.youtube.com/watch?v=VBokjWj_cEA&t=4s) - by _Sebastiaan Mathôt_
```

Comment:
Thanks @Pramochete for your contribution. While the two videos by Sebastiaan look pretty good, they are not really conference talks or programming presentations. Could you get rid of those? The thing is I don't wanna mix two different things in this list. Maybe we can have `awesome-video-podcast` or something and put that in there.